### ISSUE TITLE
Enable switching of useless long-distance roads (highways etc) in viewmode for non-car mobility (train mode, bike/walk, public).

### ISSUE THREAD

**Issue by TheOtherDoctor** [osmandapp/OsmAnd#20770]
### Describe the idea

As far as I can see, all the existing modes in OsmAnd have one trait in common: they show highways as fat dominant lines and railways as thin hard-to-see lines. When I travel by train, I typically have to walk or use a bike to get the last mile done, and I do not want to see paths in the map which are useless for me.
I'd also wish to be able to use OsmAnd for navigation tasks like this usecase, but it annoys me with the rather inappropriate (car-centric) view no matter which mode I chose.

### Expected behaviour

I want to have an easy way to display only the paths which are relevant for me in a non-car mobility scenario: in public transport mode and low zoom factor (i.e. long distance view) these would be train lines only(!), and in closest zoom all roads which have pedestrian paths; roads usable by bike are also fine, as are all roads with bus lines actually running. Main point is that highways are always useless in this usecase, so no need to see them cluttering my screen.
Similarly, when I am in bike or walk mode, I'd want to remove the highways without pedestrian or bike paths; yet I did not find a way for this, actually having those removed should be the default view in these modes. At the very least they should be reduced to thin grey faded lines, similar to the railway linetipe in car mode. Visual emphasis should corelate to relevance in usecase mode.
Consequently in public transport mode, it should be possible to have all tracks removed (or faded) which have no bus or tram running along them, and there should be no visual difference between subway and overground lines concerning stops symbols and station name display, since by default it does not matter if the boarding is above or below street level.
Simply adapt the visual style to the use value of a connection as it matched the current mobility scheme. 
As second, routing should work in "public transport" mode similar to "train" mode, which it does not for me, see below.
Long term vision is simply that I can ask OsmAnd in "pub transport" mode to "bring me from A to B in whatever combination is best considering the restrictions I define in the options". I am prepared to wait for such for some more years, but the visual fixes mentioned above should be quicker to implement. 

### Alternatives you've considered

Using combined routing with apps like transportr or öffi plus OsmAnd for the start and destination zones is possible but very time consuming and error prone.
When I try to route in "public transport" mode, it finds no route at all. When I try in "train" mode, it brings  me to the nearest train station and along the track, which helps already a lot to get a starting route, only the still car-centric view is annoying, and I have to figure out feasibility of local public transport for the last mile in another app.

### Context

I cannot believe I am the first user to be frustrated about this gap, which is very obvious to me. I looked through all of the app options and open issues but failed to find one which covers this. Possibly such a function even exists somewhere deep in the setups, then my request is to surface these.
An OsmAnd app with a genuine bike+train view could be a real game-changer for many users. :-)

**Comment by hhornburg** [comment]
Hey, 

In the longterm I try to create a [bikepacking profile](https://github.com/hhornburg/osmand-bikepacking) which gets rid of the car-centric approach of many map styles. It will however probably will be a long time until the profile is well usable.

**Comment by TheOtherDoctor** [comment]
That seems to go in a similar direction as I intended, thanks for the info. Anyway, I'd think this dissonance should be solved in standard OsmAnd (without a need to customize the app or its setup beyond what non-coder users are able and willing to handle), since to me the idea of profiles is very inconsequently realized currently; my rationale is simply that the choice of a mode should determine that I see those paths prominently which are of highest use value, in finding my way and focus on possibly interesting things on the way. For a backpacker, a highway is typically neither a path nor a sight.
Currently, the modes all look very much the same, which feels like way behind its potential.

**Comment by mdingemanse** [comment]
No-brainer really, and excellently described. OSMAnd is car-centric, which is historically understandable but unnecessary and actively frustrating in the hiking and biking profiles. I found this issue because I was looking for a way to hide highways in tunnels (!) which should be even less conspicuous in hiking mode. E.g. here's topo view of a hike inBergen, Norway that definitely did not cross a highway (which is in a tunnel at that point):

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/58be2ff7-363f-43b1-9e1c-3c08965b047c"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by sonora** [comment]
I understand the intent, but please note that highways, like e.g. also rivers, churches or water towers, are prominent orientation landmarks, even if you do not 'use' them directly for transportation.

Hence their absence from a map used to move about may, under many circumstances, be perceived as strange, if not incomplete. Even if you just have to cross a highway as a pedestrian, or see one through the window while sitting on a train.

**Comment by mdingemanse** [comment]
If you read the OP you'll note that it says:

> At the very least they should be reduced to thin grey faded lines, similar to the railway linetipe in car mode. Visual emphasis should corelate to relevance in usecase mode.

So it is not a call for outright removal but for (1) enabling users to make their own choices and (2) have sensible defaults.

My comment meanwhile is about highways _in tunnels_, which most definitely are neither landmarks nor uncrossable, and so should be invisible or at least faded in hiking and biking modes

**Comment by DmitryAlexei** [comment]
Some tunnels were hidden for pedestrians and cyclists in this issue
- https://github.com/osmandapp/OsmAnd/issues/20224

@mdingemanse Please try to change profile from "browse map" to pedestrian / cycling

**Comment by mdingemanse** [comment]
Less prominent, but definitely not hidden in pedestrian or cycling modes. Here is the same hike in Bergen showing the E16 which is in a tunnel at that point. (And hide underground objects is on.)

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/6d9c148f-1c0c-47a7-acd2-c440e0e69b5e"  width="450" height="auto"/> </td>
  </tr>
</table>

**Comment by vherrmann** [comment]
It would be best, if you could assign priorities to different paths: It is e.g. quite annoying to me when planing a cycling route that the major cycling routes aren't shown when I'm zoomed out. The same holds for public transportation.
